tests/unit/tasks/test_python_task.py

```diff
@@ -0,0 +1,43 @@
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.structures import Pipeline
+from griptape.tasks import PythonTask, BaseTask
+from tests.mocks.mock_prompt_driver import MockPromptDriver
+
+
+def hello_world(task: BaseTask) -> BaseArtifact:
+    greeting = "Hello World!"
+    return TextArtifact(greeting)
+
+
+def non_outputting(task: BaseTask) -> BaseArtifact:
+    # If your task function doesn't have an output, return task.input
+    temp = task.input
+    return temp
```

Comment:
We were just using the temp line so we could inspect in the debugger while we were working through issues. Let's get rid of the `temp = task.input` line and just `return task.input`